### PR TITLE
Update `GBC 2023 Spring` match result and podium

### PR DESCRIPTION
**Pull request by HanHan3z**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Tournaments/GBC/GBC_2023_Spring/en.md [hunk 1]
```diff
@@ -76,8 +79,107 @@ The GB Cup 2023 Spring is run by the Team GB and various community members.
...
 
+## Podium
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Spring/en.md:82]: from the template

```suggestion
## Podium

This competition has come to an end and resulted in the following podium:
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
- the badges have the wrong size. they need to be 86x40 px
- all sentences need to be in past tense (except for the ruleset)

File: wiki/Tournaments/GBC/GBC_2023_Spring/en.md [hunk 2]
```diff
@@ -40,7 +40,10 @@ The **GB Cup 2023 Spring** (***GBC 2023 Spring***) is an osu!mania 4K point-base
 | *9th place – 12th place* | 30 RMB |
 | *13th place – 16th place* | 20 RMB |
 | *17th place – 24th place* | 10 RMB |
-| *Full attendance prize* | KanonBot profile badge (all groups), 15 RMB (Group B), 25 RMB (Group A), 35 RMB (Group S), 45 RMB (Group EX) |
+| *Full attendance prize* | KanonBot profile badge (All Groups), 15 RMB (Group B), 25 RMB (Group A), 35 RMB (Group S), 45 RMB (Group EX) |
```
[the remark this answers is not in the document]
Comment (yobrevelc) [reply wiki/Tournaments/GBC/GBC_2023_Spring/en.md:43]: changed

File: wiki/Tournaments/GBC/GBC_2023_Spring/en.md [hunk 3]
```diff
@@ -40,7 +40,10 @@ The **GB Cup 2023 Spring** (***GBC 2023 Spring***) is an osu!mania 4K point-base
...
+
+![GBC 2023 Spring Banner](img/GBC-2023-Spring-badge-1st.png)\
+![GBC 2023 Spring KanonBot 1st Profile Badge](img/badge-kanon-1st.png) ![GBC 2023 Spring Banner KanonBot 2nd Profile Badge](img/badge-kanon-2nd.png) ![GBC 2023 Spring Banner KanonBot 3rd Profile Badge](img/badge-kanon-3rd.png) ![GBC 2023 Spring Banner KanonBot Attended Profile Badge](img/badge-kanon-attended.png)
```
[the remark this answers is not in the document]
Comment (yobrevelc) [reply wiki/Tournaments/GBC/GBC_2023_Spring/en.md:46]: changed

File: wiki/Tournaments/GBC/GBC_2023_Spring/en.md (hunk 1 above)
Comment (yobrevelc) [reply wiki/Tournaments/GBC/GBC_2023_Spring/en.md:82]: changed

File: wiki/Tournaments/GBC/GBC_2023_Spring/zh.md [hunk 4]
```diff
@@ -40,7 +40,9 @@ tags:
 | *季后赛第九至十二名* | 30 RMB |
 | *季后赛第十三至十六名* | 20 RMB |
 | *季后赛第十七至二十四名* | 10 RMB |
-| *常规赛全勤奖* | 猫猫定制勋章(Pending), 15 RMB(B组), 25 RMB(A组), 35 RMB(S组), 45 RMB(EX组) |
+| *常规赛全勤奖* | 猫猫定制勋章, 15 RMB(B组), 25 RMB(A组), 35 RMB(S组), 45 RMB(EX组) |
+
+![GBC 2023 Spring banner](img/GBC-2023-Spring-badge-1st.png) ![GBC 2023 Spring KanonBot 1st-place badge](img/badge-kanon-1st.png) ![GBC 2023 Spring banner KanonBot 2nd-place badge](img/badge-kanon-2nd.png) ![GBC 2023 Spring KanonBot 3rd-place badge](img/badge-kanon-3rd.png) ![GBC 2023 Spring KanonBot attendance badge](img/badge-kanon-attended.png)
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Spring/zh.md:45]: the alt text in brackets `[]` should be translated. they appear if the images fail to load and may be used by screen readers

Comment (yobrevelc) [reply wiki/Tournaments/GBC/GBC_2023_Spring/zh.md]: finished

**Review by yobrevelc — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**